src/devices/OneWire/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# OneWire
+
+## Summary
+Provide a brief description on what the component is and its functonality.
```

Comment:
please add some comment on how to connect it or set it up on linux.

Description can be really brief and other bullets can be removed, i.e.:

```
# 1-wire

Allows using 1-wire devices such as digital thermometers (i.e. DS18B20).

## How to setup 1-wire

(put description here)
```

if there is nothing needed for setup at least mention if it can be connected to any pin and if not then which one can be used

Reply:
If you (and eric) decide I should push forward on this PR, then of course I will fix comments, readme and sample.

Reply:
yes you got Eric's blessing 😄